bin/dearchive_files.py

```diff
@@ -116,6 +116,8 @@
     else:
         dearch_f = db.fetch_file(basename=path)
     
+    assert dearch_f.name is not None, f"Fetching from database is not successful. " \
```

Comment:
This can be changed to`assert dearch_f is not None` so the right AssertionError gets printed

Reply:
Thats already done in L144 I think, This is a check for the object not having the key name defined.

Reply:
The issue is when `dearch_f` is None and does not contain `name`. It is throwing the following error: `AttributeError: 'NoneType' object has no attribute 'name'` instead of `AssertionError: Fetching from database is not successful. Name of the file is not defined Can not proceed`